bc_obps/service/reporting_year_service.py

```diff
@@ -0,0 +1,7 @@
+from reporting.models.reporting_year import ReportingYear
+
+
+class ReportingYearService:
+    @classmethod
+    def get_current_reporting_year(cls) -> ReportingYear:
+        return ReportingYear.objects.get(reporting_year=2024)
```

Comment:
This will let apps retrieve the reporting year until we have a proper way of determining this